js/arrayOfCards.js

```diff
@@ -0,0 +1,263 @@
+		path:'cards/2d.png'
+	},
+	{
+		value:2,
+		suit:'hearts',
+		path:'cards/2h.png'
+	},
+	{
+		value:2,
+		suit:'spades',
+		path:'cards/2s.png'
+	},
+	{
+		value:2,
+		suit:'clubs',
+		path:'cards/2c.png'
+	},
+	{
+		value:3,
+		suit:'diamonds',
+		path:'cards/3d.png'
+	},
+	{
+		value:3,
+		suit:'hearts',
+		path:'cards/3h.png'
+	},
+	{
+		value:3,
+		suit:'spades',
+		path:'cards/3s.png'
+	},
+	{
+		value:3,
+		suit:'clubs',
+		path:'cards/3c.png'
+	},
+	{
+		value:4,
+		suit:'diamonds',
+		path:'cards/4d.png'
+	},
+	{
+		value:4,
+		suit:'hearts',
+		path:'cards/4h.png'
+	},
+	{
+		value:4,
+		suit:'spades',
+		path:'cards/4s.png'
+	},
+	{
+		value:4,
+		suit:'clubs',
+		path:'cards/4c.png'
+	},
+	{
+		value:5,
+		suit:'diamonds',
+		path:'cards/5d.png'
+	},
+	{
+		value:5,
+		suit:'hearts',
+		path:'cards/5h.png'
+	},
+	{
+		value:5,
+		suit:'spades',
+		path:'cards/5s.png'
+	},
+	{
+		value:5,
+		suit:'clubs',
+		path:'cards/5c.png'
+	},
+	{
+		value:6,
+		suit:'diamonds',
+		path:'cards/6d.png'
+	},
+	{
+		value:6,
+		suit:'hearts',
+		path:'cards/6h.png'
+	},
+	{
+		value:6,
+		suit:'spades',
+		path:'cards/6s.png'
+	},
+	{
+		value:6,
+		suit:'clubs',
+		path:'cards/6c.png'
+	},
+	{
+		value:7,
+		suit:'diamonds',
+		path:'cards/7d.png'
+	},
+	{
+		value:7,
+		suit:'hearts',
+		path:'cards/7h.png'
+	},
+	{
+		value:7,
+		suit:'spades',
+		path:'cards/7s.png'
+	},
+	{
+		value:7,
+		suit:'clubs',
+		path:'cards/7c.png'
+	},
+	{
+		value:8,
+		suit:'diamonds',
+		path:'cards/8d.png'
+	},
+	{
+		value:8,
+		suit:'hearts',
+		path:'cards/8h.png'
+	},
+	{
+		value:8,
+		suit:'spades',
+		path:'cards/8s.png'
+	},
+	{
+		value:8,
+		suit:'clubs',
+		path:'cards/8c.png'
+	},
+	{
+		value:9,
+		suit:'diamonds',
+		path:'cards/9d.png'
+	},
+	{
+		value:9,
+		suit:'hearts',
+		path:'cards/9h.png'
+	},
+	{
+		value:9,
+		suit:'spades',
+		path:'cards/9s.png'
+	},
+	{
+		value:9,
+		suit:'clubs',
+		path:'cards/9c.png'
+	},
+	{
+		value:10,
+		suit:'diamonds',
+		path:'cards/10d.png'
+	},
+	{
+		value:10,
+		suit:'hearts',
+		path:'cards/10h.png'
+	},
+	{
+		value:10,
+		suit:'spades',
+		path:'cards/10s.png'
+	},
+	{
+		value:10,
+		suit:'clubs',
+		path:'cards/10c.png'
+	},
+	{
+		value:11,
+		suit:'diamonds',
+		path:'cards/11d.png'
+	},
+	{
+		value:11,
+		suit:'hearts',
+		path:'cards/11h.png'
+	},
+	{
+		value:11,
+		suit:'spades',
+		path:'cards/11s.png'
+	},
+	{
+		value:11,
+		suit:'clubs',
+		path:'cards/11c.png'
+	},
+	{
+		value:12,
+		suit:'diamonds',
+		path:'cards/12d.png'
+	},
+	{
+		value:12,
+		suit:'hearts',
+		path:'cards/12h.png'
+	},
+	{
+		value:12,
+		suit:'spades',
+		path:'cards/12s.png'
+	},
+	{
+		value:12,
+		suit:'clubs',
+		path:'cards/12c.png'
+	},
+	{
+		value:13,
+		suit:'diamonds',
+		path:'cards/13d.png'
+	},
+	{
+		value:13,
+		suit:'hearts',
+		path:'cards/13h.png'
+	},
+	{
+		value:13,
+		suit:'spades',
+		path:'cards/13s.png'
+	},
+	{
+		value:13,
+		suit:'clubs',
+		path:'cards/13c.png'
+	}
+
+];
```

Comment:
I'm not sure, but I think it's more easier to use css sprite, like i did. In that case you needn't create such a big array manually, but just write function changing background position.